modules/@ergonode/ui/src/components/Grid/Layout/Collection/Cells/GridCollectionDefaultCell.vue

```diff
@@ -90,6 +90,14 @@ export default {
                     value: args,
                 });
             }
+            if (this.data.actions.get) {
```

Comment:
It should not emit two actions. When only GET is present is should emit only `preview`. When GET and EDIT is present it should emit `edit`